src/frontend/src/pages/CreateChangeRequestPage/CreateChangeRequestView.tsx

```diff
@@ -81,166 +69,143 @@ const CreateChangeRequestsView: React.FC<CreateChangeRequestViewProps> = ({
   crDesc,
   onSubmit,
   proposedSolutions,
-  setProposedSolutions
+  setProposedSolutions,
+  handleCancel
 }) => {
-  const { handleSubmit, control } = useForm<FormInput>({
+  const {
+    handleSubmit,
+    control,
+    formState: { errors },
+    register
+  } = useForm({
     resolver: yupResolver(schema),
-    defaultValues: { wbsNum, what: crDesc, why: [{ type: ChangeRequestReason.Other, explain: '' }] }
+    defaultValues: {
+      wbsNum,
+      what: crDesc,
+      why: [{ type: ChangeRequestReason.Other, explain: '' }],
+      type: ChangeRequestType.Issue
+    }
   });
-  const { fields, append, remove } = useFieldArray({ control, name: 'why' });
+  const { fields: whys, append: appendWhy, remove: removeWhy } = useFieldArray({ control, name: 'why' });
 
   const permittedTypes = Object.values(ChangeRequestType).filter(
     (t) => t !== ChangeRequestType.Activation && t !== ChangeRequestType.StageGate
   );
 
   return (
-    <>
-      <PageTitle title={'New Change Request'} previousPages={[{ name: 'Change Requests', route: routes.CHANGE_REQUESTS }]} />
-      <PageBlock title={''}>
-        <form id={'create-standard-change-request-form'} onSubmit={handleSubmit(onSubmit)}>
-          <Grid container spacing={2}>
-            <Grid item xs={6} md={6}>
-              <Controller
-                name="wbsNum"
-                defaultValue={wbsNum}
-                control={control}
-                rules={{ required: true }}
-                render={({ field, fieldState }) => (
+    <form
+      id={'create-standard-change-request-form'}
+      onSubmit={(e) => {
+        e.preventDefault();
+        e.stopPropagation();
+        handleSubmit(onSubmit)(e);
+      }}
+      onKeyPress={(e) => {
+        e.key === 'Enter' && e.preventDefault();
+      }}
+    >
+      <PageTitle title="New Change Request" previousPages={[{ name: 'Change Requests', route: routes.CHANGE_REQUESTS }]} />
+      <PageBlock title="Details">
+        <Grid container spacing={2}>
+          <Grid item xs={12} md={3}>
+            <Box>
+              <Typography variant="caption">WBS Number</Typography>
+            </Box>
+            <ReactHookTextField name="wbsNum" control={control} placeholder="1.1.0" errorMessage={errors.wbsNum} />
+          </Grid>
+          <Grid item xs={12} md={9}>
+            <Box>
+              <Typography variant="caption">Type</Typography>
+            </Box>
+            <Controller
+              name="type"
+              control={control}
+              rules={{ required: true }}
+              render={({ field, fieldState }) => (
+                <FormControl>
```

Comment:
can we really not do this one with the MUI select? The other case in this form makes sense, but this one seems pretty simple/ similar to other stuff we've done

Reply:
we defintely can i just wanted it to be consistent on the page